components/AddNewRoom/Form/index.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+export default function NewRoomForm({ onCreateRoom }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    event.target.reset();
+    onCreateRoom(data);
+  }
+  return (
+    <form onSubmit={handleSubmit}>
+      <StyledHeading>Add new room</StyledHeading>
+      <StyledLabel htmlFor="roomName">
+        Room Name
+        <input type="text" id="name" name="name" required></input>
+      </StyledLabel>
+      <StyledCreateButton type="submit">Create room</StyledCreateButton>
```

Comment:
you could create one button component to give them all the same styling so it would be more consistent :) 
with ternarys you could give them attributes and variants, so that you can small and big buttons for example within the one button component

Reply:
Yeah, you're absolutely right. I will create a button component in one of my upcoming US :)